### PR TITLE
chore: remove account export in onboarding

### PR DESCRIPTION
**Pull request by bumi**
People are not aware that this is sensitive data and share screenshot and videos
Export is now possible from the export function

### Screenshots of the changes (If any)
image

### CODE REVIEW

**Comment by kiwiidb** [comment]
You mean: export is "now" possible from the export function?

**Review by kiwiidb — APPROVED** [review]
LGTM

**Comment by bumi** [comment]
uuups. yes, it's possible

**Review by escapedcat — APPROVED** [review]

**Comment by bumi** [comment]
I've removed the empty success page and directly redirect to the final success page. 

@escapedcat how do I deal with removing translation strigs?

File: src/app/screens/connectors/NewWallet/index.tsx [hunk 1]
```diff
@@ -63,24 +65,22 @@ export default function NewWallet() {
...
           `${t("pre_connect.errors.create_wallet_error")} ${e.message}`
         );
-      })
-      .finally(() => {
         setLoading(false);
       });
   }
 
-  async function next(event: React.FormEvent<HTMLFormElement>) {
+  async function next(lndhub: LNDHubCreateResponse) {
```
Comment (escapedcat) [inline src/app/screens/connectors/NewWallet/index.tsx:80]: Would it be easier to understand if we call this i.e. `postSignUp` instead of next?

**Review by escapedcat — APPROVED** [review]

File: src/app/screens/connectors/NewWallet/index.tsx (hunk 1 above)
Comment (bumi) [reply src/app/screens/connectors/NewWallet/index.tsx:80]: it was next before, so I did not change it. do you care, should I change it?

Comment (escapedcat) [reply src/app/screens/connectors/NewWallet/index.tsx:80]: No, not blocking. I'll keep this in mind.

**Comment by escapedcat** [comment]
Green!

**Pull request merged**